packages/desktop-client/src/i18n.ts

```diff
@@ -28,3 +28,30 @@ i18n
       transSupportBasicHtmlNodes: false,
     },
   });
+
+const interleaveArrays = (
+  ...arrays: any[] // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
```suggestion
  ...arrays: React.ReactNode[]
```

Reply:
That type isn't quite right.  First, this is an array of arrays.  But the harder part is that I think it can be almost anything (hence why I put 'any' there).  It definitely can be more than a ReactNode - it could be a string, such as the ", and" string used to join items.

Reply:
I could try something like `(React.ReactNode | string)[][]` but I am not sure if that will cover all possible future cases.

Reply:
Hmm I was under the impression that ReactNode included anything renderable, which is what we really want to represent here? But nice catch on the array of arrays type, that totally slipped my mind

Reply:
Not sure, I'll give it a try and find out.